couch/datadog_checks/couch/couch.py

```diff
@@ -76,12 +76,16 @@ def check(self, instance):
             except Exception as e:
                 raise errors.ConnectionError("Unable to talk to the server: {}".format(e))
 
-            if version.startswith('1.'):
-                self.checker = CouchDB1(self)
-            elif version.startswith('2.'):
-                self.checker = CouchDB2(self)
-            else:
-                raise errors.BadVersionError("Unkown version {0}".format(version))
+            try:
+                major_version = int(version.split('.')[0])
+                if major_version == 0:
+                    raise errors.BadVersionError("Unknown version {}".format(version))
+                elif major_version <= 1:
+                    self.checker = CouchDB1(self)
+                else:
+                    self.checker = CouchDB2(self)
```

Comment:
is couchDB3 supported in `CouchDB2`? Could we add a comment to clarify that

Reply:
done :)